contracts/cw20-merkle-airdrop/helpers/src/airdrop.ts

```diff
@@ -0,0 +1,45 @@
+import keccak256 from 'keccak256';
+import { MerkleTree } from 'merkletreejs';
+
+class Airdrop {
+  private tree: MerkleTree;
+
+  constructor(accounts: Array<{ address: string; amount: string }>) {
+    const leaves = accounts.map((a) => keccak256(a.address + a.amount));
+    this.tree = new MerkleTree(leaves, keccak256, { sort: true });
```

Comment:
Ah, `MerkleTree` is configurable, so we could use another algorithm that is well supported in Rust and JS.

Sha2 comes to mind, as this is what is used everywhere in Cosmos, and pretty much any other linux utility